pull_request_template.md

```diff
@@ -1,13 +1,31 @@
+<!-- @formatter:off -->
+
 Why this PR is needed
-----
-Add a sentence or two on the reason this PR exists. Remember that the reviewer likely doesn't have as much context as you. Help them help you.
+---------------------
+<!-- Add a sentence or two on the reason this PR exists. Remember that the reviewer likely doesn't have as much context as you. Help them help you. -->
```

Comment:
HTML comments show up when you are filling out the PR template, but not in the rendered view that reviewers see.
Idea here is you don't have to bother to delete them.